app/views/admin/edition_tags/edit.html.erb

```diff
@@ -72,6 +72,7 @@
           }
         } %>
 
+        <%= link_to("Cancel", admin_edition_path(@edition), class: "govuk-link") %>
         <a class="govuk-link" href="<%= admin_edition_path(@edition) %>">Cancel</a>
```

Comment:
I think this needs to come out then right?

Reply:
Thanks!